Dockerfile

```diff
@@ -12,21 +12,24 @@ RUN yum install -y tigervnc-server supervisor wget java-11-openjdk-devel novnc f
 
 RUN mkdir /ideaIC-2020.2.2 && wget -qO- https://download.jetbrains.com/idea/ideaIC-2020.2.2.tar.gz | tar -zxv --strip-components=1 -C /ideaIC-2020.2.2 && \
     mkdir -p /JetBrains/IdeaIC && \
-    for f in "/JetBrains" "/ideaIC-2020.2.2" "/etc/passwd"; do \
+    mkdir /etc/default/jetbrains && \
+    for f in "/JetBrains" "/ideaIC-2020.2.2" "/etc/passwd" "/etc/default/jetbrains"; do \
       echo "Changing permissions on ${f}" && chgrp -R 0 ${f} && \
       chmod -R g+rwX ${f}; \
     done
 
-COPY --chown=0:0 /etc/entrypoint.sh /entrypoint.sh
+COPY --chown=0:0 etc/entrypoint.sh /entrypoint.sh
```

Comment:
good catch :-)

Reply:
I thought it was redundant to copy system entrypoint.sh 🙂